hack/verify-structured-logging.sh

```diff
@@ -49,12 +49,9 @@ while IFS='' read -r line; do
 done < <(cat "${migrated_packages_file}")
 
 
-# TODO: Improve concurrancy here
 ret=0
-for package in "${migrated_packages[@]}"; do
-    GOOS=linux    logcheck "$KUBE_ROOT/$package" || ret=$?
-    GOOS=windows  logcheck "$KUBE_ROOT/$package" || ret=$?
-done
+GOOS=linux    logcheck "$KUBE_ROOT/$migrated_packages" || ret=$?
```

Comment:
you need to expand the array here. Also shell check failure for the same 
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/100246/pull-kubernetes-verify/1371430538098774016#1:build-log.txt%3A2129

Reply:
Fixed it. That explains why lint finished that fast.